<ui version="4.0" >
 <class>SpectrumAlignmentDialogTemplate</class>
 <widget class="QDialog" name="SpectrumAlignmentDialogTemplate" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Spectrum alignment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Layer 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="layer_list_1" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Layer 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="layer_list_2" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Preferences</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="tolerance_label" >
          <property name="text" >
           <string>Tolerance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tolerance_spinbox" >
          <property name="maximum" >
           <double>1000000.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.010000000000000</double>
          </property>
          <property name="value" >
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="unit_group" >
        <property name="title" >
         <string>Unit</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QRadioButton" name="da" >
           <property name="text" >
            <string>Da</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ppm" >
           <property name="text" >
            <string>ppm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpectrumAlignmentDialogTemplate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpectrumAlignmentDialogTemplate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
